packages/medusa/src/api/routes/store/carts/update-cart.ts

```diff
@@ -219,6 +219,9 @@ export class StorePostCartsCartReq {
   @IsOptional()
   context?: object
 
+  @IsOptional()
+  metadata?: Record<string, unknown>
```

Comment:
todo: Add property to `@schema StorePostCartsCartReq`

Reply:
Done ✅